src/Microsoft.PowerShell.Commands.Diagnostics/ImportCounterCommand.cs

```diff
@@ -240,7 +240,7 @@ protected override void EndProcessing()
                     break;
 
                 default:
-                    Debug.Assert(false, string.Format(CultureInfo.InvariantCulture, "Invalid parameter set name: {0}", ParameterSetName));
+                    Debug.Assert(false, string.Format(CultureInfo.InvariantCulture, $"Invalid parameter set name: {ParameterSetName}"));
```

Comment:
Debug.Assert supports interpolated string handler and we can use interpolated string directly [Debug.Assert](https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Debug.cs,89)
```suggestion
                    Debug.Assert(false, $"Invalid parameter set name: {ParameterSetName}");
```